src/output_tools.h

```diff
@@ -34,6 +38,103 @@ bool ReadIdLess(const std::pair<uint32_t, MappingRecord> &a,
   return a.second.read_id_ < b.second.read_id_;
 }
 
+// The class for handling barcode convertion.
```

Comment:
Can we have a separate header named barcode_translator.h, which holds BarcodeTranslator?

Reply:
Done.